src/components/List.js

```diff
@@ -15,15 +17,26 @@ const List = ({ firestore }) => {
   };
 
   const handleChange = (e, item) => {
+    const daysInterval = secondsToDays(
```

Comment:
Great job naming functions based on their purpose but maybe some comments in this section would make it clearer exactly what's happening.